README.md

```diff
@@ -26,6 +26,31 @@ With a slight tweak of the y-axis label, core domain charts can be used to plan
 
 ![alt text](resources/architecture-migration-core-domain-chart.jpg "Architecture Migration Core Domain Chart")
 
+## Suggestions for Measuring Complexity and Differentiation
+
+Firstly, measuring complexity and differentiation of your domains is hard and is usually quite subjective. Strategy is a bet on the future, so you can never be sure exactly how complex or differentiating something will turn out to be. But there is still a lot of value in discussing and visualising your beliefs of the complexity and differentiation of each domain.
+
+The following clues can help to assess the complexity and differentiation of each domain. 
+ 
+- How difficult is it to design a conceptual model for the domain and build (and maintain) it as software? (essential domain complexity)
+- Is the current technical solution more complex than it needs to be to provide the current functionality? (accidental technical complexity)
+- Are there complex processes, calculations, and decisions happening outside the software? (operational complexity)
+- How hard would it be for a new entrant to the market to match or exceed the capability?
+- How hard would it be for an existing competitor to match or exceed the capability?
+- How much of an advantage is currently produced by the domain (revenue, brand, engagement)?
+- How much of an advantage could be produced by domain (revenue, brand, engagement)?
+- What damage would occur to the brand if major or recurring failures happened in the domain?
+- Which cynefin domain does it fall within?
+
+Complexity can manifest in different forms. Here are some clues for uncovering the essential domain complexity, accidental technical complexity, and operational complexity of each (sub)domain:
+
+- the difficulty in discovering potential value
+- designing the rules, logic, and workflows are fundamentally hard to get right
+- simpler rules and logic but operating at large scale
+- high levels of differentiation are needed to displace existing market leaders or in saturated markets
+- specialist expertise that is difficult and expensive to acquire 
```

Comment:
I would rather formulate whole sentences for each clue
```suggestion
- it is difficult to discover the potential value
- designing the rules, logic, and workflows are fundamentally hard to get right
- the rules and logic are simpler but operating at a large scale is needed
- high levels of differentiation are needed to displace existing market leaders or in saturated markets
- it is required specialist expertise that is difficult and expensive to acquire 
```

Reply:
I agree. I was being a bit lazy. Full sentences are better like the section above.